cmake/modules/extensions.cmake

```diff
@@ -1559,6 +1559,71 @@ function(zephyr_syscall_header_ifdef feature_toggle)
   endif()
 endfunction()
 
+# Verify blobs fetched using west. If the sha256 checksum isn't valid, a warning/
+# fatal error message is printed (depends on REQUIRED flag).
+#
+# Usage:
+#   zephyr_blobs_verify(MODULE module [FILES file [files...]] [REQUIRED])
```

Comment:
if `FILES` is given, then there is no reason for `MODULE` to be required argument.

A relative path argument is relative to `CMAKE_CURRENT_SOURCE_DIR`, so the module name is not used for path expansion. Also the module name isn't used later in `if(NOT "A ${path}" IN_LIST BLOBS_LIST)` and `{abspath}` ensures that all blobs have unique paths.